algraph.py

```diff
@@ -0,0 +1,54 @@
+
+class ALGraph:
+    """adjacency_list graph"""
+
+    def __init__(self):
+        self.adjacency_list = {}
```

Comment:
Where do you keep track of ALL of your vertices/nodes?  (What if a vertex/node has no edges associated with it?  e.g. person who just joined Facebook and hasn't added any friends yet.)

Reply:
It's just a toy graph to allow for implementing the findTopologicalOrdering algorithm.